plugins/in_kafka/in_kafka.c

```diff
@@ -428,6 +446,11 @@ static struct flb_config_map config_map[] = {
     0, FLB_TRUE, offsetof(struct flb_in_kafka_config, buffer_max_size),
     "Set the maximum size of chunk"
    },
+   {
+    FLB_CONFIG_MAP_BOOL, "enable_auto_commit", FLB_IN_KAFKA_ENABLE_AUTO_COMMIT,
```

Comment:
I don't think so we need to this config option as this is included in the  `rdkafka.enable.auto.commit`